package.json

```diff
@@ -63,6 +63,7 @@
   },
   "dependencies": {
     "@poppinss/manager": "^2.1.2",
+    "aws-sdk": "^2.573.0",
```

Comment:
AWS SDK must be a peer dependency, since not everyone would want to use SES for sending emails